stories/Button.stories.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Button from '../src/Button';
+import Icon from '../src/Icon';
+
+const stories = storiesOf('components/Button', module);
+
+stories.addParameters({
+  info: {
+    text: `There are 3 main button types described in material design. The raised button is a standard button that signify actions and seek to give depth to a mostly flat page. The floating circular action button is meant for very important functions. Flat buttons are usually used within elements that already have depth like cards or modals.`
+  }
+});
+
+stories.add('Default', () => (
+  <div>
+    <Button waves="light" style={{ marginRight: '5px' }}>
+      button
+    </Button>
+    <Button waves="light" style={{ marginRight: '5px' }}>
+      button<Icon left>cloud</Icon>
+    </Button>
+    <Button waves="light">
+      button<Icon right>cloud</Icon>
+    </Button>
+  </div>
+));
+
+stories.add('Floating', () => (
+  <Button floating large className="red" waves="light" icon="add" />
+));
+
+stories.add('Flat', () => (
+  <Button flat waves="light">
+    Button
+  </Button>
+));
+
+stories.add('Submit', () => (
+  <Button type="submit" waves="light">
+    Submit<Icon right>send</Icon>
+  </Button>
+));
+
+stories.add('Large', () => (
+  <div>
+    <Button node="a" waves="light" large style={{ marginRight: '5px' }}>
+      Button
+    </Button>
+    <Button node="a" waves="light" large style={{ marginRight: '5px' }}>
+      Button<Icon left>cloud</Icon>
+    </Button>
+    <Button node="a" waves="light" large>
+      Button<Icon right>cloud</Icon>
+    </Button>
+  </div>
+));
+
+stories.add('Small', () => (
+  <div>
+    <Button node="a" waves="light" small style={{ marginRight: '5px' }}>
+      Button
+    </Button>
+    <Button node="a" waves="light" small style={{ marginRight: '5px' }}>
+      Button<Icon left>cloud</Icon>
+    </Button>
+    <Button node="a" waves="light" small>
+      Button<Icon right>cloud</Icon>
+    </Button>
+  </div>
+));
+
+stories.add('Disabled', () => (
+  <div>
+    <Button large disabled style={{ marginRight: '5px' }}>
+      Button
+    </Button>
+    <Button disabled style={{ marginRight: '5px' }}>
+      Button
+    </Button>
+    <Button flat disabled style={{ marginRight: '5px' }}>
+      Button
+    </Button>
+    <Button floating disabled>
+      <Icon>add</Icon>
+    </Button>
+  </div>
+));
+
+stories.add('Link', () => (
+  <Button waves="light" node="a" href="http://react-materialize.github.io/">
+    Open Me In New Tab
+  </Button>
+));
+
+stories.add('Top FAB', () => (
```

Comment:
let's categorize the fab stories for a neeter storybook, try to add `stories.add('FAB/Top'` Bottom, etc...

Reply:
Like this?

![screen shot 2019-01-12 at 15 49 26](https://user-images.githubusercontent.com/1959694/51076610-bd0f4e80-1681-11e9-86b2-e92912743baf.png)